packages/svelte-cloudinary/src/config.ts

```diff
@@ -0,0 +1,108 @@
+import { VERSION as SVELTE_CLOUDINARY_VERSION } from './version';
+import { VERSION as SVELTE_VERSION } from 'svelte/compiler';
+import { setContext, getContext } from 'svelte';
+import { defu } from 'defu';
+import type {
+	AnalyticsOptions,
+	ConfigOptions,
+} from '@cloudinary-util/url-loader';
+
+const STORE_KEY = 'svelte-cloudinary-v2-config';
+
+export const DEFAULT_ANALYTICS: AnalyticsOptions = Object.freeze({
```

Comment:
it doesnt look like the analytics is working properly, this is from the docs

<img width="529" alt="image" src="https://github.com/user-attachments/assets/fb35a80d-1a98-4404-a3eb-6f137ed7108a">

Reply:
https://github.com/cloudinary-community/svelte-cloudinary/issues/128